### PR TITLE
Fixed card errors and typos

### PR DESCRIPTION
**Pull request by haec0007**
Fixes issues #30 #31 #32 #33 #35

### CODE REVIEW

File: starwars/sets/TOR.txt [hunk 1]
```diff
@@ -81,7 +81,7 @@ Resource Shortage	TOR	TOR025_Resource_Shortage	N	Mission		X				R	25		Choose one:
...
 Revan (D)	TOR	TOR105_Revan_D	L	Character	Old Republic Jedi Master	6	50	5	5	C	105		Pay 2 Force -> Revan gets +1 power and Critical Hit 1 for this attack. | Pay 2 Force -> Evade 2	
-Revan's Mask	TOR	TOR028_Revan_s_Mask_A	N	Equipment	Character - Armor Defense	1				R	28		Pay 1 build point, 1 Force -> Equip | Equipped Jedi or Sith gets +1 health | Equipped Darth Revan or Revan also gets "When the battle phase starts, you may choose an ability on another card in Revan's stack. If you do, Revan gets the chosen ability until end of turn."	
+Revan's Mask (A)	TOR	TOR028_Revan_s_Mask_A	N	Equipment	Character - Armor Defense	1				R	28		Pay 1 build point, 1 Force -> Equip | Equipped Jedi or Sith gets +1 health | Equipped Darth Revan or Revan also gets "When the battle phase starts, you may choose an ability on another card in Revan's stack. If you do, Revan gets the chosen ability until end of turn."
```
Comment (ChrisMoExp) [inline starwars/sets/TOR.txt:84]: this will stop Revan's Mask loading from people's decks. Can you take this one out?

**Review by ChrisMoExp — CHANGES_REQUESTED** [review]
i think they're all good excep the Revan's Mask one. The Vong one probably doesn;t affect many decks, if any. But the mask is likely in tournament decks.

File: starwars/sets/TOR.txt (hunk 1 above)
Comment (haec0007) [reply starwars/sets/TOR.txt]: I've reverted the Revan's Mask fix and moved it to its own pull request (#50). This pull request now fixes only issues #30 #31 #32 #33

**Review by ChrisMoExp — APPROVED** [review]
yeah

**Pull request merged**